src/src/ESP32_WebUpdate.cpp

```diff
@@ -159,7 +159,8 @@ void BeginWebUpdate()
     server.on(
         "/update", HTTP_POST, []() {
       server.sendHeader("Connection", "close");
-      server.send(200, "text/plain", (Update.hasError()) ? "FAIL" : "OK");
+      server.send(200, "text/plain", (Update.hasError()) ? "FAIL" : "Update completed.");
+      delay(200);
```

Comment:
You should really call `server.client().stop();`
As this will flush and close the connection to the client, and guarantee that the message has been sent.